lighthouserc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  ci: {
+    collect: {
+      url: ['http://localhost:3000/'],
```

Comment:
Could we collect mutiple urls via configuration ? (homepage, article page...etc)

Reply:
Address in [150a7d8](https://github.com/twreporter/twreporter-react/pull/2227/commits/150a7d824f944af5a57cd26a3dde4ab85931705c).